openfga/v1/openfga_service.proto

```diff
@@ -731,6 +745,107 @@ message ListObjectsResponse {
   ];
 }
 
+message ListUsersRequest {
+  string store_id = 1 [
+    json_name = "store_id",
+    (validate.rules).string = {pattern: "^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$"},
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"01YCP46JKYM8FJCQ37NMBYHE5X\""}
+  ];
+
+  string authorization_model_id = 2 [
+    json_name = "authorization_model_id",
+    (validate.rules).string = {
+      pattern: "^[ABCDEFGHJKMNPQRSTVWXYZ0-9]{26}$",
+      ignore_empty: true
+    },
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"01G5JAVJ41T49E9TT3SKVS7X1J\""}
+  ];
+
+  Object object = 3 [
+    json_name = "object",
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"document:example\""},
+    (google.api.field_behavior) = REQUIRED
+  ];
+
+  string relation = 4 [
+    (validate.rules).string = {pattern: "^[^:#@\\s]{1,50}$"},
+    (google.api.field_behavior) = REQUIRED,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "\"reader\""}
+  ];
+
+  repeated ListUsersFilter user_filters = 5 [
+    json_name = "user_filters",
+    (google.api.field_behavior) = REQUIRED,
+    (validate.rules).repeated = {
+      min_items: 1,
+      max_items: 1
+    },
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {example: "[{\"type\": \"user\"}, {\"type\": \"group\", \"relation\": \"member\"}]"}
+  ];
+
+  openfga.v1.ContextualTupleKeys contextual_tuples = 6 [json_name = "contextual_tuples"];
```

Comment:
I know existing APIs use `openfga.v1.ContextualTupleKeys`, but I thought I'd ask to see if there was interest to change this to the more normalized definition. Having the `openfga.v1.ContextualTupleKeys` stutters. Should just be:

```
repeated TupleKey contextual_tuples = 6 ...;
```

Reply:
No objections. Is the intention to make this change to the rest of the endpoints or will this be inconsistent with the rest?

Reply:
The intention is to just make the change here. It impacts the gRPC API and makes it stutter less. We wouldn't change existing signatures so it would break existing gRPC clients, but those using ListUsers after we release it won't suffer the same stutter.

As for the HTTP API, it would unimpactful if we changed it across the board.

Reply:
Made this change and it cleaned-up a lot of code relating to contextual tuples. 👍